src/axis.js

```diff
@@ -1,5 +1,12 @@
-c3_chart_internal_fn.initAxis = function () {
-    var $$ = this, config = $$.config, main = $$.main;
+function _Axis(owner) {
+    API.call(this, owner);
+}
+
+inherit(API, _Axis);
+
+_Axis.prototype.init = function init() {
```

Comment:
will do, but is there anything special reason why we use something like 'fn', instead of using ***.prototype directly?